### ISSUE TITLE
a question

### ISSUE THREAD

**Issue by mikeyjhom** [taldatech/soft-intro-vae-pytorch#20]
Dear Mr. Tal Daniel
    I apologize for taking up your time. I have a question and would like to request your help. I trained the soft-intro_vae model using the code you provided, and the image generated during the training process was very good. However, after the model training was completed, I used the trained model to generate images, but received strange results
   This is the image generated during the training process

![FXF8PMII9)~LRL@0)IWFABF](https://github.com/taldatech/soft-intro-vae-pytorch/assets/164597567/145ad634-ba3d-46ae-af99-869af44cd8a8)

    This is the image generated by calling the trained model
![mmexport1711957926710](https://github.com/taldatech/soft-intro-vae-pytorch/assets/164597567/0082382f-5316-4803-83c5-2b750ff23b11)
![image_2](https://github.com/taldatech/soft-intro-vae-pytorch/assets/164597567/a6542c7d-f19a-4da9-8d33-65fbdf2bfdae)
   Is the method I used to call the trained model incorrect?
   Think about seeking your advice on a solution. I hope to receive your help. Thank you very much

**Comment by taldatech** [comment]
Hmmmm I think it has to do with Batch Normalization. I assume you use the standard architecture (and not the style-based one). Try turning BatchNorm on/off (`model.train()/model.eval()`) and see how it affects the results. The architecture in this model is very out-dated. I would replace all the Batch Normalization with Group Normalization to avoid depending on the batch size statistics.

**Comment by mikeyjhom** [comment]
Okay, thank you very much for your reply. I will try to solve the problem based on your suggestions
